.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -22,7 +22,7 @@ body:
           required: false
         - label: Would you like to work on this issue?
           required: false
-        - label: Join our slack channel using [wg-manifests](https://www.kubeflow.org/docs/about/community/).
+        - label: Join our CNCF slack channel using [kubeflow-platform](https://www.kubeflow.org/docs/about/community/).
```

Comment:
please use [#kubeflow-platfrom](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2) to link directly to the channel and add the link to https://www.kubeflow.org/docs/about/community/ as well